ecchronos-binary/src/test/behave/features/steps/ecctool_trigger_repair_steps.py

```diff
@@ -17,41 +17,71 @@
 from behave import when, then  # pylint: disable=no-name-in-module
 from ecc_step_library.common_steps import match_and_remove_row, validate_header, validate_last_table_row  # pylint: disable=line-too-long
 
+TABLE_ROW_FORMAT_PATTERN = r'\| .* \| {0} \| {1} \| (COMPLETED|IN_QUEUE|WARNING|ERROR) \| \d+[.]\d+ \| .* \|'
```

Comment:
would like to have this separate from run-repair (even though we are duplicating). It will be easier to remove.